cmd/wazero/wazero_test.go

```diff
@@ -285,13 +297,27 @@ func TestRun(t *testing.T) {
 <== (err=<nil>,ok=true)
 `, bearMode, bearMtime),
 		},
+		{
+			name:       "GOARCH=wasm GOOS=js hostlogging=crypto and filesystem",
+			wasm:       wasmCat,
+			wazeroOpts: []string{"--hostlogging=crypto", "--hostlogging=filesystem"},
```

Comment:
we only need to test combination once. this was the easiest place since `GOARCH=wasm GOOS=js` initializes with crypto functions.